janitor/io.py

```diff
@@ -2,6 +2,7 @@
 import subprocess
 from glob import glob
 from io import StringIO
+from openpyxl import load_workbook
```

Comment:
Two questions regarding requirements:
1. Can we make this import optional?
2. I suppose we need to add this to our package requirements? And I suppose this package needs to be optional as well. To prevent pyjanitor install from being too bloated.

Reply:
which brings us to #826 ; any ideas on how to go about this?

Reply:
I'll probably have a look at this and see where the rabbit hole leads

Reply:
Good link there, @samukweku!

Reply:
For 1., I was thinking of shifting this openpyxl import into the `xlsx_table` function itself. Then do a try import + raising import error message similar to how `biology.py` does it.

For 2. I'm not sure if `openpyxl` fits into any existing EXTRA_REQUIRES categories ([cf. here](https://github.com/pyjanitor-devs/pyjanitor/blob/dev/setup.py#L59)), maybe a new category like `io` or `excel` would be necessary. I'm leaning more towards `io`to leave the option open in the future for more io functions that require other packages.